### PR TITLE
Cleanup mailmap, and update authors

### PR DESCRIPTION
**Pull request by thaJeztah**
**- What I did**

Updated the authors file, and cleaned-up the `.mailmap` file (standardise on format, and sort alphabetically to make it easier to maintain)

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @vdemeester PTAL

**Review by vdemeester — APPROVED** [review]
LGTM 🐮

File: AUTHORS [hunk 1]
```diff
@@ -1663,6 +1696,7 @@ Thomas Grainger <tagrain@gmail.com>
 Thomas Hansen <thomas.hansen@gmail.com>
 Thomas Leonard <thomas.leonard@docker.com>
 Thomas LEVEIL <thomasleveil@gmail.com>
+Thomas Léveil <thomasleveil@gmail.com>
```
Comment (thaJeztah) [inline AUTHORS]: Oh; need to fix this, I see what happened

**Review by thaJeztah — COMMENTED** [review]
fixed duplicate entry.

author LGTM (CI is not testing this)

**Pull request merged**